app/services/health_monitor.py

```
@@ -0,0 +1,372 @@
+                if exchange not in exchange_health:
+                    exchange_health[exchange] = {
+                        "health_status": "UNKNOWN",
+                        "circuit_breaker_state": "UNKNOWN",
+                        "failure_count": 0,
+                        "success_count": 0
+                    }
+                
+                # Add last check timestamp
+                exchange_health[exchange]["last_check"] = datetime.utcnow().isoformat()
+        
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)